### DISCUSSION TITLE
Eelevation data in Routes resp. Tracks

### DISCUSSION THREAD

**Discussion by GuentherGredy** [osmandapp/OsmAnd#12294] [Q&A]
Dear OsmAnd Support, going down my learning curve using Garmin BaseCamp for planning and OsmAnd for navigating in the field, I learned, that producing **routes** in BaseCamp is much easier than creating **tracks**. In consequence I exported these **routes** to OsmAnd folder "import" and had a lot of fun navigating them. Now I'm interested in elevation data and found out, that from a route file I cannot retrieve the curve along the route, while from a track file this is possible. Do you have any idea why this is the case? I have attached screen shots and corresponding route resp. track files. Thanks a lot in advance. Günther

[Transfer.zip](https://github.com/osmandapp/OsmAnd/files/6894573/Transfer.zip)

**Comment by vshcherb** [comment]
With ```<rtept>``` track you need to calculate route between points in any case (so elevation data will be attached once calculated), with ```<trkpt>``` it is not necessary but it's possible. You can use 'Attach To Roads' functionality and in that case elevation data will be provided with recalculated route
